src/components/layout/Tabs.js

```diff
@@ -86,11 +96,17 @@ const Li = styled.li`
       : ''};
 `
 const A = styled.a`
-  ${props => `border-bottom: 3px solid ${selectTypeColor(props.name)}`};
+  ${props =>
+    `border-bottom: 3px ${selectBorderStyle(props.name)} ${selectTypeColor(
+      props.name
+    )}`};
   ${props => {
     if (
       props.isActive &&
-      (props.name === PART_TIME || props.name === GRAPHQL_BOOTCAMP)
+      (props.name === GRAPHQL_BOOTCAMP ||
+        props.name === GRAPHQL_API ||
+        props.name === GRAPHQL_CLIENT ||
+        props.name === GRAPHQL_WORKSHOP)
     ) {
```

Comment:
GRAPHQL_API and GRAPHQL_CLIENT will be removed once those dates pass and we deprecated those types. My plan is to search and strip those from the code base at that time.